pkg/util/github/branch_test.go

```diff
@@ -7,14 +7,14 @@ import (
 )
 
 type newBranchTest struct {
-	baseTest
+	BaseTest
```

Comment:
Why should this struct be exported? Will it be used in other pkg? @jxs1211

Reply:
It's unnecessary, fixed